x-pack/plugins/security/server/config_deprecations.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ConfigDeprecationProvider } from 'src/core/server';
+
+export const securityConfigDeprecationProvider: ConfigDeprecationProvider = ({
```

Comment:
**note** with the exception of the last deprecation in this array, this was simply lifted from `index.ts` in order to facilitate unit testing